src/Concordium/Client/Runner.hs

```diff
@@ -1859,17 +1859,19 @@ checkAndGetContractReceiveName contrInfo receiveName = do
     then return receiveName
     else
       if not $ CI.hasFallbackReceiveSupport contrInfo
-        then do
-          logWarn [[i|The contract does not have an entrypoint named '#{receiveName}' and no fallback entrypoint.|]]
-          confirmed <- askConfirmation $ Just "Do you still want to continue? "
-          unless confirmed $ logFatal ["aborting..."]
-          return receiveName
+        then confirm_abort_or_continue
```

Comment:
The convention in Haskell, at least in our project, is to use lower camel case for variable names.

Reply:
And ideally the message in this case, and in the case below, would be slightly different.

Here the reason is that they are calling a V0 contract, below it is that they are calling a V1 contract which opts to not have a fallback.

Reply:
what about this one?